### PR TITLE
Terminate immediately if compilation fails.

### PR DESCRIPTION
**Pull request by Wilfred**
Fixes https://github.com/rejeep/evm/issues/35.

### CODE REVIEW

File: lib/evm/system.rb [hunk 1]
```diff
@@ -5,7 +5,11 @@ def initialize(executable)
     end
 
     def run(*args)
-      Kernel.system(@executable, *args)
+      succeeded = Kernel.system(@executable, *args)
+      unless succeeded
+        print "Failed! See logs above for error."
```
Comment (rejeep) [inline lib/evm/system.rb]: Can use please use `puts` here (to get a newline) and use single quotes for consistency.

**Comment by Wilfred** [comment]
@rejeep fixed and pushed a new version.

**Comment by rejeep** [comment]
@Wilfred Thanks! :+1:

**Comment by rexim** [comment]
Yay! Thank you @Wilfred! :)

**Pull request closed without merging**